docs/_includes/variation-content.html

```diff
@@ -40,7 +40,7 @@ <h1 class="component-header u-mb30">
     {% endfor %}
 
     {%- if jump_links %}
-    <ul class="m-list m-list__horizontal u-mb30">
+    <ul class="m-list m-list__horizontal m-list__horizontal-spaced">
```

Comment:
Oops, I see one mistake in the tweaks I gave. You originally had `u-mb30` here, which gives 30px of bottom margin. `m-list__horizontal-spaced` gives 15px of bottom margin. So if 15px is fine then we're all good, but if not, you'd change the CSS to (also probably would make sense to include the `px`):

```
.m-list__horizontal-spaced {
    margin-bottom: unit( 30px / @base-font-size-px, em );
}

.m-list__horizontal-spaced > .m-list_item {
    margin-bottom: unit( 15px / @base-font-size-px, em );
}
```

Reply:
I defer to you and the designers (@sonnakim) about this. It looks like the original implementation from @contolini had 30px ([here](https://github.com/cfpb/design-system/blame/99625d8c0c838b6bc71097d798756fa667152bd0/docs/_includes/variation-content.html#L30)), so maybe we should stick with that?

Reply:
I've pushed a commit that changes this to 30px.